osmo_jupyter/db_access.py

```diff
@@ -0,0 +1,109 @@
+import textwrap
+from getpass import getpass
```

Comment:
Description here (or elsewhere) would be helpful.

Reply:
you mean, a description of what the file does?
    """ Functions to access node data from the Osmo database
    """
sound good?

Reply:
It wasn't immediately clear what and why the getpass function was being used.

Reply:
gah, I replied to the notification e-mail and the reply didn't get threaded in in the obvious way. I said:
"oh, good point. I'm gonna comment where it's used though, not the import."